src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,77 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+    .grid.primary > .content > .ui.secondary.pointing.menu {
+        margin-top: 0;
+    }
+
+    .grid.primary > .topbar{
+        grid-area: topbar;
+    }
+ 
+    .grid.primary > .content{
+        padding: @padding;
+        grid-area: content;
+        background: #fff;
+        border: 1px solid #eaeaea;
+        border-radius: 3px;
+    }
+    .grid.primary > .footer {
+        grid-area: footer;
+        display: flex;
+        flex-wrap: wrap;
+        padding: 1em;
```

Comment:
Use the common @padding variable since 1em = 16px in our implementation

Reply:
Let me do that